{
  "networkCallRecords" : [ {
    "Method" : "PUT",
    "Uri" : "https://azstoragesdkaccount.file.core.windows.net/setmetadataerrorshareasyncapitestssetmetadataerror80f896065?restype=share&comp=metadata",
    "Headers" : {
      "x-ms-version" : "2019-02-02",
      "User-Agent" : "azsdk-java-azure-storage-file-share/12.0.0-beta.6 (11.0.4; Windows 10 10.0)",
      "x-ms-client-request-id" : "34ee2b3a-b28c-4fd9-960e-ce76e18098aa"
    },
    "Response" : {
      "x-ms-version" : "2019-02-02",
      "Server" : "Windows-Azure-File/1.0 Microsoft-HTTPAPI/2.0",
      "x-ms-error-code" : "ShareNotFound",
      "retry-after" : "0",
      "Content-Length" : "217",
      "StatusCode" : "404",
      "x-ms-request-id" : "0ec73572-501a-0025-6939-aaa8c5000000",
      "Body" : "ï»¿<?xml version=\"1.0\" encoding=\"utf-8\"?><Error><Code>ShareNotFound</Code><Message>The specified share does not exist.\nRequestId:0ec73572-501a-0025-6939-aaa8c5000000\nTime:2019-12-04T00:25:09.4534228Z</Message></Error>",
      "Date" : "Wed, 04 Dec 2019 00:25:08 GMT",
      "x-ms-client-request-id" : "34ee2b3a-b28c-4fd9-960e-ce76e18098aa",
      "Content-Type" : "application/xml"
    },
    "Exception" : null
  } ],
  "variables" : [ "setmetadataerrorshareasyncapitestssetmetadataerror80f896065" ]
}